buildenv/jenkins/testJobTemplate

```diff
@@ -124,8 +124,10 @@ println "JDK_VERSIONS: ${JDK_VERSIONS}"
 println "GROUPS: ${GROUPS}"
 println "ARCH_OS_LIST: ${ARCH_OS_LIST}"
 
-def OPENJ9_REPO = "https://github.com/eclipse-openj9/openj9.git"
-def ADOPTOPENJDK_REPO = "https://github.com/AdoptOpenJDK/openjdk-tests.git"
+
+def OPENJ9_REPO = "https://github.com/eclipse/openj9.git"
```

Comment:
The original value `https://github.com/eclipse-openj9/openj9.git` is correct. Please do not change it.